[16364] Fullscreen create

furinvader opened the pull request:
Create a work package in the fullscreen view.
This PR adds the green add-work-package button back again with its intended functionality.

**Work Package**
https://community.openproject.org/work_packages/16364/activity

If this PR is merged, the following bug is also fixed:
https://community.openproject.org/work_packages/21651/activity

furinvader committed:
Add route for fullscreen create

furinvader committed:
Change full-create state url

furinvader committed:
Add work package create button to fullscreen vew

furinvader committed:
Rename WP create route file

furinvader committed:
Get projectIdentifier from $stateParams in WP-create form

furinvader committed:
Make directive scope attributes optional

TeatroIO commented:
I've prepared a stage to preview changes. <a href='http://furinvader-features-16364-fullscreen-create.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com' target='_blank'>Open stage</a> or <a href='http://furinvader-features-16364-fullscreen-create.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com/teatro.log' target='_blank'>view logs</a>.

furinvader committed:
Remove details class from WP-create form

furinvader committed:
Adjust action bar position

furinvader committed:
Implement success state attribute

It contains the state, where the user will
be redirected to after te WP is updated.

furinvader committed:
Move "go back on cancel" method to WP new controller

furinvader committed:
Stop edit mode when leaving create form page

furinvader committed:
Fix WP routing error

oliverguenther commented on frontend/app/routing.js:
```diff
@@ -34,11 +34,12 @@ angular.module('openproject')
-  $urlRouterProvider.when('/work_packages/{id}', function ($match) {
-    if($match.id.length === 0) {
-      return '/work_packages';
+  $urlRouterProvider.when('/work_packages/{id: [0-9]}', function ($match) {
```
This must be changed to `.when('/work_packages/{id:[0-9]+}', ...`, or it won't match any work package.

furinvader committed:
Hide WP-add button when in edit mode

[ci skip]

furinvader committed:
Add alternative template for WP-create form

furinvader committed:
Add details view class to default WP-create form

This fixes the toolbar position.

mk84 committed:
Fix alignment of description field in full create

mk84 committed:
Fix actions toolbar position in fullcreate

mk84 committed:
Make edit button inside toolbar blue

furinvader committed:
Set details view button as active if new WP is being created

mk84 committed:
Add missing toolbar-items class

mk84 committed:
Use absolute layout mode on wp full create

furinvader committed:
Merge pull request #3 from mk84/22043-wp-full-create-ui

[22043] WP fullcreate UI improvments
